lyon 0.16 does not build.

I ran `cargo update` and now my project doesnt build because lyon is failing to build.
I can reproduce the issue with an otherwise empty project as well.

Might be related to the 0.17 release earlier today, https://github.com/nical/lyon/pull/617 but I dont see the 0.17 release on crates.io yet.

There are 255 errors like:
```
error[E0432]: unresolved imports `crate::geom::math`, `crate::AttributeStore`, `crate::ControlPointId`, `crate::EndpointId`, `crate::Event`, `crate::IdEvent`, `crate::PathEvent`, `crate::PositionStore`
  --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/lib.rs:56:9
   |
56 | pub use crate::geom::math;
   |         ^^^^^^^^^^^^^^^^^ no `math` in the root
   | 
  ::: /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/path.rs:8:13
   |
8  | use crate::{AttributeStore, ControlPointId, EndpointId, Event, IdEvent, PathEvent, PositionStore};
   |             ^^^^^^^^^^^^^^  ^^^^^^^^^^^^^^  ^^^^^^^^^^  ^^^^^  ^^^^^^^  ^^^^^^^^^  ^^^^^^^^^^^^^

error[E0412]: cannot find type `Point` in this scope
  --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/builder.rs:98:31
   |
98 |     fn move_to(&mut self, to: Point);
   |                               ^^^^^ not found in this scope
   |
help: consider importing one of these items
   |
77 | use crate::geom::Point;
   |
77 | use lyon_geom::Point;
   |

error[E0412]: cannot find type `Point` in this scope
   --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/builder.rs:101:31
    |
101 |     fn line_to(&mut self, to: Point);
    |                               ^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
77  | use crate::geom::Point;
    |
77  | use lyon_geom::Point;
    |

error[E0412]: cannot find type `Point` in this scope
   --> /home/rukai/.cargo/registry/src/github.com-1ecc6299db9ec823/lyon_path-0.16.2/src/builder.rs:109:35
    |
109 |     fn current_position(&self) -> Point;
    |                                   ^^^^^ not found in this scope
    |
help: consider importing one of these items
    |
77  | use crate::geom::Point;
    |
77  | use lyon_geom::Point;
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "lyon"
-version = "0.17.0"
+version = "0.17.1"
 description = "2D Graphics rendering on the GPU using tessellation."
 authors = [ "Nicolas Silva <nical@fastmail.com>" ]
 repository = "https://github.com/nical/lyon"
@@ -37,11 +37,11 @@ profiling = ["lyon_tessellation/profiling"]
 
 [dependencies]
 
-lyon_tessellation = { version = "0.17.0", path = "tessellation/" }
-lyon_algorithms = { version = "0.17.0", path = "algorithms/" }
-lyon_extra = { version = "0.17.0", optional = true, path = "extra/" }
-lyon_svg = { version = "0.17.0", optional = true, path = "svg/" }
-lyon_tess2 = { version = "0.17.0", optional = true, path = "tess2/" }
+lyon_tessellation = { version = "0.17.1", path = "tessellation/" }
+lyon_algorithms = { version = "0.17.1", path = "algorithms/" }
+lyon_extra = { version = "0.17.1", optional = true, path = "extra/" }
+lyon_svg = { version = "0.17.1", optional = true, path = "svg/" }
+lyon_tess2 = { version = "0.17.1", optional = true, path = "tess2/" }
 
 [workspace]
 members = [

--- a/algorithms/Cargo.toml
+++ b/algorithms/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "lyon_algorithms"
-version = "0.17.0"
+version = "0.17.1"
 description = "2D Path manipulation/transformation algorithms."
 authors = [ "Nicolas Silva <nical@fastmail.com>" ]
 repository = "https://github.com/nical/lyon"
@@ -20,6 +20,6 @@ serialization = ["serde", "lyon_path/serialization"]
 
 [dependencies]
 
-lyon_path = { version = "0.17.0", path = "../path" }
+lyon_path = { version = "0.17.1", path = "../path" }
 serde = { version = "1.0", optional = true, features = ["serde_derive"] }
 sid = "0.6"

--- a/extra/Cargo.toml
+++ b/extra/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "lyon_extra"
-version = "0.17.0"
+version = "0.17.1"
 description = "Various optional utilities for the lyon crate."
 authors = [ "Nicolas Silva <nical@fastmail.com>" ]
 repository = "https://github.com/nical/lyon"
@@ -14,5 +14,5 @@ name = "lyon_extra"
 
 [dependencies]
 
-lyon_path = { version = "0.17.0", path = "../path" }
-lyon_svg = { version = "0.17.0", path = "../svg" }
+lyon_path = { version = "0.17.1", path = "../path" }
+lyon_svg = { version = "0.17.1", path = "../svg" }

--- a/geom/Cargo.toml
+++ b/geom/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "lyon_geom"
-version = "0.16.3"
+version = "0.17.0"
 description = "2D quadratic and cubic bézier arcs and line segment math on top of euclid."
 authors = ["Nicolas Silva <nical@fastmail.com>"]
 repository = "https://github.com/nical/lyon"

--- a/path/Cargo.toml
+++ b/path/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "lyon_path"
-version = "0.17.0"
+version = "0.17.1"
 description = "Types and utilities to store, build and iterate over 2D paths."
 authors = [ "Nicolas Silva <nical@fastmail.com>" ]
 repository = "https://github.com/nical/lyon"
@@ -16,5 +16,5 @@ name = "lyon_path"
 serialization = ["serde", "lyon_geom/serialization"]
 
 [dependencies]
-lyon_geom = { version = "0.16.3", path = "../geom" }
+lyon_geom = { version = "0.17.0", path = "../geom" }
 serde = { version = "1.0", optional = true, features = ["serde_derive"] }

--- a/svg/Cargo.toml
+++ b/svg/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "lyon_svg"
-version = "0.17.0"
+version = "0.17.1"
 description = "SVG helpers for the lyon crates."
 authors = [ "Nicolas Silva <nical@fastmail.com>" ]
 repository = "https://github.com/nical/lyon"
@@ -14,6 +14,6 @@ path = "src/lib.rs"
 
 [dependencies]
 
-lyon_path = { version = "0.17.0", path = "../path" }
+lyon_path = { version = "0.17.1", path = "../path" }
 
 svgtypes = "0.5.0"

--- a/tess2/Cargo.toml
+++ b/tess2/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "lyon_tess2"
-version = "0.17.0"
+version = "0.17.1"
 description = "An additional path tessellator for lyon using libtess2."
 authors = [ "Nicolas Silva <nical@fastmail.com>" ]
 repository = "https://github.com/nical/lyon"
@@ -20,6 +20,6 @@ serialization = ["serde", "lyon_tessellation/serialization"]
 
 [dependencies]
 
-lyon_tessellation = { version = "0.17.0", path = "../tessellation" }
+lyon_tessellation = { version = "0.17.1", path = "../tessellation" }
 serde = { version = "1.0", optional = true, features = ["serde_derive"] }
 tess2-sys = "0.0.1"

--- a/tessellation/Cargo.toml
+++ b/tessellation/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 
 name = "lyon_tessellation"
-version = "0.17.0"
+version = "0.17.1"
 description = "A low level path tessellation library."
 authors = [ "Nicolas Silva <nical@fastmail.com>" ]
 repository = "https://github.com/nical/lyon"
@@ -23,10 +23,10 @@ profiling = []
 
 [dependencies]
 
-lyon_path = { version = "0.17.0", path = "../path" }
+lyon_path = { version = "0.17.1", path = "../path" }
 sid = "0.6"
 serde = { version = "1.0", optional = true, features = ["serde_derive"] }
 arrayvec = "0.5"
 
 [dev-dependencies]
-lyon_extra = { version = "0.17.0", path = "../extra" }
+lyon_extra = { version = "0.17.1", path = "../extra" }

